LBHackney-IT/lbh-fss-portal-frontend
# Feature/list all services

**Description**
- whilst logged in as an admin user, be able to see a list of all services 
- whilst logged in as a VCSO user, when adding a new service, if navigating back to a previous page my answers are preserved 
- whilst logged in as a VCSO user, when editing an existing service, the service form is pre-populated with info 
- whilst logged in either as an admin or a VCSO user, have the ability to delete a service

**Test instructions**

- pull this branch (you will automatically be logged in as an admin user)
- navigate to **/services** and check you can see a list of services 
- you should be able to remove a service using 'Remove' from the action dropdown 
- you should be able to edit a service by clicking on the service name (check if clicking on the name it takes you to the service form, however dont try to proceed through the service form as it is likely the mock location data will prevent you progressing past the location page) 
- switch to a VCSO user - you can do this by navigating to **mock-api/account/GET.js** and comment out the 'ADMIN' section and uncomment the 'VCSO - WITH services and WITH organisation' section immediately below it
- navigate to **/service**
- click 'Add service' and progress through the form adding information on each page, check when navigating back to previous pages via the navigation bar the information persist 
- click on the existing services, proceed through the form checking information has been pre-populated with info, please review against the raw data returned by the API by looking at the first service 'Barrows, Homenick and Jerde' listed in **mock-api/mockServices.json**

**Tasks**
https://app.clickup.com/t/9e5u8x
https://app.clickup.com/t/9e5u91
